build.assets/Dockerfile-multiarch-devtoolset-gcc

```diff
@@ -0,0 +1,98 @@
+FROM centos:7 AS base
+
+# Automatically supplied by the Docker buildkit
+ARG TARGETARCH
+
+# Minimal build tool versions
+ARG GCC_VERSION=7.4.0
```

Comment:
The same as above, we can probably switch to something never, but if this is going to take too much time, I don't mind the current version.